FAQs.md

```diff
@@ -0,0 +1,121 @@
+## FREQUENTLY ASKED QUESTIONS
+
+This is the list of Frequently asked questions for Jockey music player. It is intended to serve as a self-service utility document. This is to aid new users or already existing users of Jockey music player.
+This document may be updated from time to time to suit any new changes to the software as new developments or changes will be implemented.
+
+1. **What is Jockey Music Player?**
+
+Jockey is a music player for Android based on Google's Material Design standards. The point of Jockey is to be a simple, lightweight media player with features designed for music enthusiasts and casual listeners alike. 
+
+
+2. **What are the features of jockey music player?**
+
+Jockey music player is designed with the essential features of a music player and they include;
+
+- playlist
+- theme customization
+- equalizer
+- sleep timer
+- search tool
```

Comment:
I think some of these are pretty expected, and this list excludes some features that Jockey has that a lot of other music players don't. Here's the section from Google Play for reference (but don't feel obligated to copy it verbatim):

> **Features**
•  Customizable interface with 81 color combinations to choose from
•  Last.fm integration for retrieving artist images and descriptions
•  Smart Playlists with fully customizable rules
•  Custom equalizers
•  Gapless playback
•  Home screen widgets
And of course, every feature you'd expect from a music player including playlist editing, lockscreen controls and artwork, and fluid navigation between all of your content

Reply:
alright I'll include these.

Reply:
I think I should copy as it is here though. there's actually no point changing words if they still mean same things and it'll be good to mention all the features that's stated on google playstore